tests/docker/docker-compose.yml

```diff
@@ -30,6 +30,12 @@ services:
     - redpanda-test
   rp:
     image: vectorized/redpanda-test-node
+    deploy:
+      resources:
+        limits:
+          cpus: '0.75'
```

Comment:
@gousteris how did we come to these numbers? The changes looks okay to me but not sure if we should settle with these numbers.